src/core/src/bootstrap/Constants.py

```diff
@@ -189,8 +189,9 @@ class TelemetryEventLevel(EnumBackport):
         LogAlways = "LogAlways"
 
     TELEMETRY_TASK_NAME = "ExtensionCoreLog"
-    TELEMETRY_AT_AGENT_NOT_COMPATIBLE_ERROR_MSG = "The minimum Azure Linux Agent version prerequisite for Linux patching was not met. Please update the Azure Linux Agent on this machine following instructions here: http://aka.ms/UpdateLinuxAgent"
-    TELEMETRY_AT_AGENT_COMPATIBLE_MSG = "The minimum Azure Linux Agent version prerequisite for Linux patching was met."
+
+    TELEMETRY_AT_AGENT_NOT_COMPATIBLE_ERROR_MSG = "Minimum Linux Agent version prerequisite not met. Resolution: http://aka.ms/UpdateLinuxAgent"
```

Comment:
'Azure Linux Agent' instead of 'Linux Agent' everywhere...

Reply:
Adding Azure truncates the error message in status blob

Reply:
"Unsupported older Azure Linux Agent version. To resolve: http://aka.ms/UpdateLinuxAgent" - 87 chars vs. 92 in PR.
 
The compatible message is less relevant but is short anyway so can have Azure in there.